Project/Assets/ML-Agents/Examples/Dodgeball/Scripts/AgentCubeMovement.cs

```diff
@@ -208,10 +207,20 @@ void FixedUpdate()
                 //                print("AddFallingForce");
 
             }
+
             if (!allowHumanInput)
             {
                 return;
             }
+
+            var rotate = m_DodgeballInput.rotateInput;
+            if (!m_Bp.IsInHeuristicMode() && rotate.magnitude > 0.00001)
```

Comment:
This print will only print the rotation of the agent using the Neural Network if its magnitude is pretty much greater than 0